tests/common/util/TestObject.cpp

```diff
@@ -89,6 +89,8 @@ createLedgerHeader(std::string_view ledgerHash, ripple::LedgerIndex seq, std::op
     ledgerHeader.seq = seq;
 
     if (age) {
+        // Note: be cautios of using small age values as the underlying NetClock precision is seconds
+        // and the small time difference may lead to comparison bugs
```

Comment:
Maybe we should add `ASSERT()` here?

Reply:
Sorry, I don't get it, what will we assert?

Reply:
Assert that the value is big enough, but if it should be different from other values, we could't add an assertion here.

Reply:
Yep, that's exactly why I was asking. So, the best we could do is to write a good comment.